### PR TITLE
Active Gate plugins installation

### PR DESCRIPTION
**Pull request by luigibk**
### JIRA link (if applicable) ###

[AB#1084]

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by naikajah — APPROVED** [review]

File: modules/dynatrace-activegate/vm.tf [hunk 1]
```diff
@@ -24,13 +24,22 @@ data "azurerm_key_vault_secret" "ssh_public_key" {
   key_vault_id = data.azurerm_key_vault.subscription_vault.id
 }
 
+data "azurerm_key_vault_secret" "dynatrace_plugin_storage_key" {
```
Comment (timja) [inline modules/dynatrace-activegate/vm.tf:27]: shouldn't this be a data source on the storage account for the subscription why is it coming from key vault?

Comment (luigibk) [reply modules/dynatrace-activegate/vm.tf:27]: As the plugins are the same for all envs, it's a single storage account as the source for all installations (and envs). Unless I misunderstand your question that is :-)

Comment (timja) [reply modules/dynatrace-activegate/vm.tf:27]: I think you misunderstand

I mean why do you need to read this from key vault you can just read it as a datasource on the storage account?

Comment (luigibk) [reply modules/dynatrace-activegate/vm.tf]: 👍

**Review by timja — APPROVED** [review]

**Pull request merged**